components/Modals/BatchTransfer/index.tsx

```diff
@@ -0,0 +1,343 @@
+                        {selectedCoin.map((val: any, i) => (
+                            <div key={i} className={styles.wrapperInput}>
+                                <code className={styles.code}>{val.value}</code>
+                                <input
+                                    className={styles.inputCurrency}
+                                    type="text"
+                                    placeholder={t('Type amount')}
+                                    onChange={(event) => {
+                                        setSavedCurrency(false)
+                                        updateDonation(null)
+                                        const value = event.target.value
+                                        inputCoin[i] = val.value + '@' + value
+                                        setInputCoin(inputCoin)
+                                    }}
+                                />
+                            </div>
+                        ))}
+                        {selectedCoin.length > 0 && (
+                            <div
+                                onClick={saveCurrency}
+                                className={!savedCurrency ? 'continueBtn' : ''}
+                                style={{ width: 'fit-content' }}
+                            >
+                                {!savedCurrency ? (
+                                    <Image src={ContinueArrow} alt="arrow" />
+                                ) : (
+                                    <div
+                                        style={{
+                                            marginTop: '5px',
+                                        }}
+                                    >
+                                        <Image
+                                            width={40}
+                                            src={TickIco}
+                                            alt="tick"
+                                        />
+                                    </div>
+                                )}
+                            </div>
+                        )}
+                        {savedCurrency && (
+                            <>
+                                <Donate />
+                                {donation !== null && (
+                                    <div className={styles.wrapperBtn}>
+                                        <div
+                                            onClick={handleSubmit}
+                                            className={
+                                                isLight
+                                                    ? 'actionBtnLight'
+                                                    : 'actionBtn'
+                                            }
+                                        >
+                                            {isLoading ? (
+                                                <ThreeDots color="yellow" />
```

Comment:
everything related to balances can be blue

Reply:
done